misc/python/materialize/checks/scenarios.py

```diff
@@ -70,15 +70,17 @@ def run(self) -> None:
         # Configure implicitly for cloud scenarios
         if not isinstance(actions[0], StartMz):
             actions.insert(0, ConfigureMz(self))
-        for action in actions:
-            action.execute(self.executor)
-            action.join(self.executor)
+
+        for id, action in enumerate(actions):
             # Implicitly call configure to raise version-dependent limits
             if isinstance(action, StartMz) or isinstance(
                 action, ReplaceEnvironmentdStatefulSet
             ):
-                ConfigureMz(self).execute(self.executor)
```

Comment:
@nrainer-materialize here, we called `execute()` on one instance of `ConfigureMz` but `join()` on another ...